### ISSUE TITLE
Gah. Your tab just crashed.

### ISSUE THREAD

**Issue by accetto** [accetto/xubuntu-vnc#3]
Firefox 68.0 and also 68.0.1 crash a lot if if opening multiple tabs.

**Comment by accetto** [comment]
Seems to be a bug in the Firefox version **68.0** and also **68.0.1**.

Saddly, the previously working fix by adding the following settings does not seem to help any more:

```javascript
user_pref("browser.tabs.remote.autostart", false);
user_pref("browser.tabs.remote.autostart.2", false);
```

I've tested both ways of installation - from the ubuntu distribution and also from the Mozilla distribution (explicit version installation), by it behaves the same in all cases.

If I'll find a solution, I'll promptly update all containers.

For now, I've mitigated the problem by keeping the previous version **67.0.4**, which does not crash at all.

A minor annoyance is that Firefox keep asking for updates (see #2). I'll try to mitigate that in the next release.

**Comment by accetto** [comment]
Reopening the issue until a fix for the versions **68.0+** will be found or a non-crashing Firefox version will be released.

**Comment by accetto** [comment]
Fixed in the version 19.08.2.

The cause of the trouble was, that since Firefox 68.0 Mozilla removed the possibility to disable multi-process through user preferences. However, it is still possible to use the environment variable MOZ_FORCE_DISABLE_E10S.

Unfortunatelly, the Firefox multi-process feature causes heavy tab crashes in Docker containers.
